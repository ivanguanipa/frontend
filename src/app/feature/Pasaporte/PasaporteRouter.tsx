import * as React from 'react';
import { Route, Switch } from 'react-router-dom';
import { LazyFallback } from 'app/shared/components/LazyFallback';

const MainPage = React.lazy(() => import('./pages/Main'));
const ShowPage = React.lazy(() => import('./pages/Show'));

export const PasaporteRouter = () => (
  <React.Suspense fallback={<LazyFallback />}>
    {/* Layout compartido entre las rutas va aquÃ­ */}
    <Switch>
      <Route path="/pasaportes" exact component={MainPage}></Route>
      <Route path="/pasaportes/show/:id" component={ShowPage}></Route>

    </Switch>
  </React.Suspense>
);
