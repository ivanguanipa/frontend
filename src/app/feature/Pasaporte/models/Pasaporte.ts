type Nullable<T> = T | null ;
export interface Pasaporte {
    id?: Nullable<number>,
    fullname: string,
    address: string,
    birthdate: string,
    documentId: number,
    applicationDate: string,
    appointmentDate?: Nullable<string>,
    amount?: Nullable<number>,
}