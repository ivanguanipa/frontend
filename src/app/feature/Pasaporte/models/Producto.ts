export interface Producto {
  title: string;
  slug: string;
  body: string;
  createdAt: string;
  updatedAt: string;
}
