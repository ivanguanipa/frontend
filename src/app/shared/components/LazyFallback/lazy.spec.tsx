import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import { Pasaporte } from 'app/feature/Pasaporte/models/Pasaporte';
import {LazyFallback} from './index'

describe('Lazy Page Test', () => {
    let componentWrapper: ShallowWrapper;
    afterEach(() => {
            componentWrapper.unmount();
        });

    it('should match snapshot', () => {

        componentWrapper = shallow(
            <LazyFallback />
        );
        expect(componentWrapper).toMatchSnapshot();
      });

})