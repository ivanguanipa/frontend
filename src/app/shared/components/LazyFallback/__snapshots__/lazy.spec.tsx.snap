// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lazy Page Test should match snapshot 1`] = `
<styled.div>
  <styled.span>
    Cargando p√°gina...
  </styled.span>
</styled.div>
`;
